packages/palette/src/elements/ProgressDots/ProgressDots.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react"
+import styled, { css } from "styled-components"
+import { color, space } from "../../helpers"
+import { Box, BoxProps } from "../Box"
+import { VisuallyHidden } from "../VisuallyHidden"
+
+const DOT_COLOR_MS = 250
+
+const Dot = styled(Box).attrs({ m: 0.5 })<{ active: boolean }>`
+  width: ${space(0.5)}px;
+  height: ${space(0.5)}px;
+  border-radius: 50%;
+  background-color: ${color("black10")};
+  transition: background-color ${DOT_COLOR_MS}ms;
+
+  ${({ active }) =>
+    active &&
+    css`
+      background-color: ${color("black100")};
+    `}
+`
+
+Dot.displayName = "Dot"
```

Comment:
❤️ 

We should check to make sure we're doing this everywhere else... really does help debugging.

Reply:
Isn't there a babel plugin for this?

Reply:
https://github.com/artsy/palette/blob/master/packages/palette/.babelrc#L16 should do it

Reply:
I guess it's not being applied within Jest — the reason I added it here is so I could target it in Jest without exporting.